src/screenComponents/mineSweeper.h

```diff
@@ -0,0 +1,35 @@
+/** An implementation of mineSweeper for use as a hacking minigame.
+ *  Original implementation by https://github.com/daid
+ */
+
+#ifndef MINESWEEPER_H
+#define MINESWEEPER_H
+
+#include "miniGame.h"
+#include "gui/gui2_togglebutton.h"
+#include <vector>
+#include <memory>
```

Comment:
I don't see anything from vector or memory being used in this header file?

Reply:
Oops. Leftovers from initially implementing the entire structure here before it was moved to the MiniGame class. Good catch, thank you.